pyanaconda/modules/common/structures/payload.py

```diff
@@ -215,3 +216,261 @@ def included_packages(self) -> List[Str]:
     @included_packages.setter
     def included_packages(self, included_packages: List[Str]):
         self._included_packages = included_packages
+
+
+class PackagesConfigurationData(DBusData):
+    """Structure for the configuration of packages."""
+
+    def __init__(self):
+        self._core_group_enabled = True
+        self._default_environment_enabled = False
+        self._environment = ""
+        self._groups = []
+        self._groups_package_types = {}
+        self._excluded_groups = []
+        self._packages = []
+        self._excluded_packages = []
+        self._docs_excluded = False
+        self._weakdeps_excluded = False
+        self._missing_ignored = False
+        self._broken_ignored = False
+        self._languages = RPM_LANGUAGES_ALL
+        self._multilib_policy = MULTILIB_POLICY_BEST
+        self._timeout = DNF_DEFAULT_TIMEOUT
+        self._retries = DNF_DEFAULT_RETRIES
+
+    @property
+    def core_group_enabled(self) -> Bool:
+        """Should the core group be installed?
+
+        :return: True if the core group should be installed
+        :rtype: bool
+        """
+        return self._core_group_enabled
+
+    @core_group_enabled.setter
+    def core_group_enabled(self, value: Bool):
+        self._core_group_enabled = value
+
+    @property
+    def default_environment_enabled(self) -> Bool:
+        """Should a default environment be installed?
+
+        :return: True if the default package set should be installed
+        :rtype: bool
+        """
+        return self._default_environment_enabled
+
+    @default_environment_enabled.setter
+    def default_environment_enabled(self, value: Bool):
+        self._default_environment_enabled = value
+
+    @property
+    def environment(self) -> Str:
+        """The packages environment marked for installation.
+
+        :return: a full environment name (as given in the comps.xml file)
+        :rtype: str
+        """
+        return self._environment
+
+    @environment.setter
+    def environment(self, value: Str):
+        self._environment = value
+
+    @property
+    def groups(self) -> List[Str]:
+        """A list of package groups marked for installation.
```

Comment:
If I understand this correctly, this basically comes from the kickstart packages section, right ? I that case it might be good to note that this might also contain module specification, as they use the same notation in kickstart as groups, for example:

```
@nodejs:6/default
@django:1.6/default
```

I think just noting that in the docstring would be enough, no need to rename the properties IMHO.

Reply:
Good point. I will have to think it through, so it might be fixed in another pull request.

Reply:
Please add FIXME keyword than.

Reply:
I have updated the doc strings and added examples.